### PR TITLE
ceph.in: fallback to install path of "ceph" when "ceph-conf" is not found

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/ceph.in [hunk 1]
```diff
@@ -503,7 +504,14 @@ def format_help(cmddict, partial=None):
 
 
 def ceph_conf(parsed_args, field, name):
-    args = ['ceph-conf']
+    cmd = 'ceph-conf'
+    bindir = os.path.dirname(__file__)
```
[the remark this answers is not in the document]
Comment (tchaikov) [reply src/ceph.in:497]: yes, i don't want to reference a substitution variable here for the `bindir`. and this assumption looks reasonable to me.

[the remark this answers is not in the document]
Comment (rzarzynski) [reply src/ceph.in:497]: Yeah, I share the impression. Maybe other reviewers will point out something but I see no obvious contraindication for doing that.

**Comment by tchaikov** [comment]
```
test_pool_update_metadata (tasks.mgr.dashboard.test_pool.PoolTest)
2020-06-16 11:37:45,048.048 INFO:__main__:----------------------------------------------------------------------
2020-06-16 11:37:45,048.048 INFO:__main__:Traceback (most recent call last):
2020-06-16 11:37:45,048.048 INFO:__main__:  File "/home/jenkins-build/build/workspace/ceph-dashboard-pr-backend/qa/tasks/mgr/dashboard/test_pool.py", line 333, in test_pool_update_metadata
2020-06-16 11:37:45,048.048 INFO:__main__:    self._validate_pool_properties(properties, self._get_pool(pool_name))
2020-06-16 11:37:45,048.048 INFO:__main__:  File "/home/jenkins-build/build/workspace/ceph-dashboard-pr-backend/qa/tasks/mgr/dashboard/test_pool.py", line 119, in _validate_pool_properties
2020-06-16 11:37:45,048.048 INFO:__main__:    self.assertEqual(health['status'], 'HEALTH_OK', msg='health={}'.format(health))
2020-06-16 11:37:45,048.048 INFO:__main__:AssertionError: 'HEALTH_WARN' != 'HEALTH_OK'
2020-06-16 11:37:45,049.049 INFO:__main__:- HEALTH_WARN
2020-06-16 11:37:45,049.049 INFO:__main__:+ HEALTH_OK
2020-06-16 11:37:45,049.049 INFO:__main__: : health={'status': 'HEALTH_WARN', 'checks': [{'severity': 'HEALTH_WARN', 'summary': {'message': 'Reduced data availability: 1 pg inactive, 1 pg peering', 'count': 2}, 'detail': [{'message': 'pg 11.10 is stuck peering since forever, current state peering, last acting [1,3,0]'}], 'muted': False, 'type': 'PG_AVAILABILITY'}], 'mutes': []}
2020-06-16 11:37:45,049.049 INFO:__main__:
2020-06-16 11:37:45,049.049 INFO:__main__:----------------------------------------------------------------------
```

**Comment by tchaikov** [comment]
jenkins test dashboard backend

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-06-19_01:26:25-rados-wip-kefu2-testing-2020-06-19-0046-distro-basic-smithi/

- cbt failures should be addressed by #35683
- cephadm failures are caused by DoS of docker.io

**Pull request merged**